resfinder/4.1.11/Dockerfile

```diff
@@ -0,0 +1,132 @@
+FROM ubuntu:focal as app
+
+# ARG sets environment variables during the build stage
+ARG RESFINDER_VER="4.1.11"
+ARG KMA_VER="1.4.3"
+
+# LABEL instructions tag the image with metadata that might be important to the user
+# Optional, but highly recommended
+LABEL base.image="ubuntu:focal"
+LABEL dockerfile.version="1"
+LABEL software="resfinder"
+LABEL software.version=${RESFINDER_VER}
+LABEL description="Identifies acquired resistance genes and point mutations in total or partial sequenced isolates of bacteria."
+LABEL website="https://bitbucket.org/genomicepidemiology/resfinder/src/master/"
+LABEL license="https://bitbucket.org/genomicepidemiology/resfinder/src/master/README.md"
+LABEL maintainer="Curtis Kapsak"
+LABEL maintainer.email="kapsakcj@gmail.com"
+
+# so that apt doesn't ask for a region during apt install step
+ARG DEBIAN_FRONTEND noninteractive
+
+# RUN executes code during the build
+# Install dependencies via apt-get or yum if using a centos or fedora base
+# ncbi-blast+ is v2.9.0
+RUN apt-get update && apt-get install -y --no-install-recommends \
+ wget \
+ python3-pip \
+ ncbi-blast+ \
+ gzip \
+ procps \
+ libz-dev \
+ git \
+ build-essential && \
+ rm -rf /var/lib/apt/lists/* && apt-get autoclean
+
+# download ResFinder code; checkout 4.1.11 tag
+RUN git clone https://bitbucket.org/genomicepidemiology/resfinder.git && \
```

Comment:
```
wget https://bitbucket.org/genomicepidemiology/resfinder/get/${RESFINDER_VER}.tar.gz
```
looks better and you don't need to install git.

Reply:
Agreed, but I think `git` will be required since I'm checking out specific commits for the pointfinder and resfinder databases.

Also didn't realize bitbucket offered a way to directly download tags, so thank you for that info

Reply:
I'll update this line, but I don't see any other way of changing the `git clone` lines for the databases. They don't have clearly tagged versions for the dbs. There a few tags, but they look to be old and missing some of the recently additions/edits to the databases

Reply:
Yeah, I haven't found a better solution that what you had. 

Is it smaller when you download the zipped file as opposed to cloning in the repo?